src/web/core/src/test/java/org/geoserver/web/systemstatus/SystemStatusMonitorPanelTest.java

```diff
@@ -29,15 +28,15 @@ public void setupTests() {
     @Test
     public void testLoad() throws Exception {
         tester.assertRenderedPage(StatusPage.class);
-        tester.clickLink("tabs:tabs-container:tabs:2:link", true);
+        tester.clickLink("tabs:tabs-container:tabs:3:link", true);
```

Comment:
Why this change?

Reply:
Test not working due to extra injected tab on testing for web/core